Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'webpacker',  '5.2.1'
 gem 'turbolinks', '5.2.1'
 gem 'jbuilder',   '2.10.0'
 gem 'bootsnap',   '1.7.2', require: false
+gem "haml"
```

Comment:
Always specify the version of your gems. 
By specifying the gem version you are locking the gem. If your gem is unlocked you can have issues with the compatibility between gems.